lib/Views/PopupMessage.html

```diff
@@ -1,7 +1,7 @@
 <div class="modal-background" data-bind="click: closeIfClickOnBackground">
     <div class="popup-message" data-bind="style: { width: width + 'px', height: height + 'px', 'max-width': maxWidth + 'px', 'max-height': maxHeight + 'px' }">
         <div class="modal-header">
-            <div class="modal-close-button" data-bind="click: close">&times;</div>
+            <div class="modal-close-button" data-bind="click: close, hasFocus: true">&times;</div>
```

Comment:
Thanks Kevin!

1. My current approach is to not have keyboard access to modal window(so it's not possible to navigate to the (i) button on the Data Catalogue tab) , and if a modal window opens up because of an error, the close button is auto focused so user can close it(this seems to be broken will fix this). But I guess we should make the modal window Accessible by keyboard as well. i will do more work on this. 

2. On national map, i think when page load, the data catalog tap get activated immediately, and that causes the hidden input box below it get focused, that's why you can't go to info tab. I think if you try it on Aremi it should work. Or if you manually focus on the now viewing tab then you should be able to tap to remove. I will double check.

Thanks for the Comments Kevin. I think I need to do more testing on this, especially with all the modal windows.